modules/overview-of-openshift-gitops.adoc

```diff
@@ -0,0 +1,39 @@
+[id="overview-of-openshift-gitops"]
+= Overview of OpenShift GitOps
+include::modules/common-attributes.adoc[]
+include::modules/gitops-document-attributes.adoc[]
+:context: overview-of-openshift-gitops
+toc::[]
+
+:FeatureName: OpenShift GitOps
+include::modules/technology-preview.adoc[leveloffset=+1]
+
```

Comment:
Use the first 2 paragraphs here to first explain what is GitOps - Maybe a small module - About GitOps https://docs.openshift.com/container-platform/4.6/architecture/cicd_gitops.html#cicd_gitops_methodology

This sentence could be added at the start of this paragraph on line 11: Red Hat Openshift GitOps is a declarative way to implement continuous deployment for cloud knative applications.